extensions/amp-access/0.1/test/test-amp-access.js

```diff
@@ -690,7 +659,9 @@ describes.fakeWin('AccessService pingback', {
     ampdoc = env.ampdoc;
     document = win.document;
     clock = sandbox.useFakeTimers();
-
+    // Polyfill performance.now() method as it is not available in fakeWin
+    // environment.
+    win.performance = {now: () => Date.now()};
```

Comment:
Since it ended up needing to be polyfilled in so many places, does it make sense to move this polyfill to `env.win` so each test doesn't need to repeat the polyfill?

Reply:
Thanks for your feedback Kris. Will actually commented on that as well and I made the changes and moved this to fake-dom.js